v0.5/recommendation/README.md

```diff
@@ -0,0 +1,262 @@
+# MLPerf Inference Benchmarks for Recommendation Task
+
+This is the reference implementation for MLPerf Inference benchmarks.
+
+### Supported Models
+
+| model | framework | accuracy | AUC | dataset | trained  | precision | notes |
```

Comment:
Which model is going to be "the official one"?

Reply:
What are the minimum hardware requirements for each model (number of GPUs, DRAM)?

Reply:
I have added a comment specifying that the largest model ~100GB is the official model, while interface to others is only provided for debugging purposes. Also, added the typical machine memory requirements.

Reply:
Fantastic, thanks Maxim!